src/components/AppMapDetailsObj.ts

```diff
@@ -504,6 +512,12 @@ export default class AppMapDetailsObj extends AppMapDetailsBase<MapMarkerObj | M
       let ll = objs.map((obj: any) => [obj.data.Translate[2], obj.data.Translate[0]]);
       let line = L.polyline(ll, { color: '#cccccc', weight: 1.5 }).addTo(map.m);
       this.korokMarkers.push(line);
+    } else if (this.obj && this.obj.korok_type == "Moving Lights") {
```

Comment:
At this point it's probably worth adding

```js
if (!this.obj)
  return;
```

at the top of this function since this is checked in every single if statement.